### PR TITLE
2.0

### PR DESCRIPTION
**Pull request by wujastyk**
I made the change content -> contents in various files.

### CODE REVIEW

File: documentation/biblatex-manuscripts-philology.tex [hunk 1]
```diff
@@ -65,26 +71,41 @@ \subsection{Mandatory}
...
+\fielditem{collection}{literal} the collection in the library. For example: 
+\verb+Supplément grec+ or \verb+Vaidyaka+.
 
-\fielditem{location}{literal} the city or place where the manuscript is kept. For example: \verb+Paris+ or \verb+Oxford+.
+\fielditem{location}{literal} the city or place where the manuscript is kept. For 
+example: \verb+Paris+ or \verb+Oxford+ or \verb+Varanasi+.
 
-\fielditem{library}{literal} the library where the manuscript is kept. For example: \verb+Bibliothèque Nationale de France+.
+\fielditem{library}{literal} the library where the manuscript is kept. For example: 
+\verb+Bibliothèque Nationale de France+ or \verb+Sarasvatī Bhavan Library+.
```
Comment (maieul) [inline documentation/biblatex-manuscripts-philology.tex]: We should correct to "Bibliothèque nationale de France"

File: documentation/biblatex-manuscripts-philology.tex [hunk 2]
```diff
@@ -65,26 +71,41 @@ \subsection{Mandatory}
...
 
-\fielditem{bookpagination}{key} the pagination of the manuscript which is studied. The standard pagination keys are allowed, but the package adds a new key: \texttt{folio}. The value of this field is to be used for printing the \bibfield{pages} and \bibfield{pagetotal}.
+\fielditem{bookpagination}{key} the pagination or foliation of the manuscript 
+which is studied. The standard \biblatex pagination keys are allowed, but the 
+package adds a new key: \texttt{folio}. 
+
+The value of this field will be used
+when printing the \bibfield{pages} and \bibfield{pagetotal}.
+E.g., ``ff.\ 3--5'' (when  \bibfield{bookpagination} =
+“folio”) or ``pp.\ 88--99'' (“page”).  See the \biblatex manual for details.
```
Comment (maieul) [inline documentation/biblatex-manuscripts-philology.tex]: I would use `~` instead of `\`

File: documentation/biblatex-manuscripts-philology.tex [hunk 3]
```diff
@@ -65,26 +71,41 @@ \subsection{Mandatory}
...
 
 \fielditem{columns}{integer} the number of column by pages. Basically, only two numbers are allowed: \verb+1+ or \verb+2+.
 
-\fielditem{dating}{literal} the dating of the manuscript. It can be for example a century.
+\fielditem{dating}{literal} the dating of the manuscript. It can be for example a 
+century.  
+
+With non-Gregorian calendars, enter the converted Gregorian date here, and the 
+date in the original era in the \bibfield{annotation} field.
```
Comment (maieul) [inline documentation/biblatex-manuscripts-philology.tex]: Is it the best way? As the`dating` field is is literal, we may add the two forms in the same place. In another way, we could create a specific field. Annotation field is normally for long commentary. 

Alos, we could look on biblatex recent change in order to see if they can manage two parallel calendar system.

File: documentation/biblatex-manuscripts-philology.tex [hunk 4]
```diff
@@ -98,17 +119,26 @@ \subsection{Optional}
 \fielditem{shortlibrary}{literal} the abreviated form of the library. Not used by the default style.
 
 
-\fielditem{pages}{range} the pages which are studied in the manuscript. In this field, you can use the macros \cs{recto} and \cs{verso}.
+\fielditem{pages}{range} the pages which are studied in the manuscript. In this 
+field, you can use the macros \cs{recto} and \cs{verso}.  This value will be used 
+when formatting citations.
+
+Use this field also for manuscript folio numbers.  E.g., “3\cs{verso}--5\cs{recto}.”
```
Comment (maieul) [inline documentation/biblatex-manuscripts-philology.tex]: Use \verb or \csquotes but not directl ""

File: documentation/biblatex-manuscripts-philology.tex [hunk 5]
```diff
@@ -118,7 +148,7 @@ \subsection{Optional}
   \item \texttt{palm-leaf};
   \item \texttt{papyrus};
   \item \texttt{paper};
-  \item \texttt{parchment}\footnote{Historically, was \texttt{pergament}, which is kept only for ascendant compatibility.}.
+  \item \texttt{parchment}.
```
Comment (maieul) [inline documentation/biblatex-manuscripts-philology.tex]: please, kept the footnote.

File: documentation/biblatex-manuscripts-philology.tex [hunk 6]
```diff
@@ -139,7 +169,10 @@ \subsection{Special}\label{fields:special}
   catalog = {[prenote1][postnote1]{key1}[prenote2][postnote2]{key2}}
 \end{minted}
 
-\fielditem{content}{literal} description of the manuscript's content, can be used instead of \bibfield{annotation}. If you want to add paragraphs inside it, you must use the \cs{par} command between each paragraph.
+\fielditem{content}{literal} description of the manuscript's content, can be used 
+with or instead of \bibfield{annotation}. It starts a new section of the description 
+output. If you want to add paragraphs inside it, you must use the \cs{par} 
+command between each paragraph.  E.g., ``Covers chapters 1--3 only.''
```
Comment (maieul) [inline documentation/biblatex-manuscripts-philology.tex]: I don't understand " It starts a new section of the description  output"

File: documentation/biblatex-manuscripts-philology.tex (hunk 1 above)
Comment (wujastyk) [reply documentation/biblatex-manuscripts-philology.tex]: I agree.

File: documentation/biblatex-manuscripts-philology.tex (hunk 2 above)
Comment (wujastyk) [reply documentation/biblatex-manuscripts-philology.tex]: I agree.

File: documentation/biblatex-manuscripts-philology.tex (hunk 3 above)
Comment (wujastyk) [reply documentation/biblatex-manuscripts-philology.tex]: Yes.  I was wrong.  One should give the date as stated in the manuscript, followed by the Gregorian date in square brackets.  See the conventions of the Anglo-American Cataloguing Rules 2, section 1.4F1 (https://archive.org/stream/pdfy-VV-4_m1HVoyDFFrA/AACR2#page/n57).
I haven't looked at the latest biblatex documentation today, but in fact having two fields - one literal and one controlled - is quite a good idea, to help with sorting.

File: documentation/biblatex-manuscripts-philology.tex (hunk 4 above)
Comment (wujastyk) [reply documentation/biblatex-manuscripts-philology.tex]: Sorry!  I wasn't able to compile the document, so errors like this slipped through my fingers.

File: documentation/biblatex-manuscripts-philology.tex (hunk 5 above)
Comment (wujastyk) [reply documentation/biblatex-manuscripts-philology.tex]: As you wish.

File: documentation/biblatex-manuscripts-philology.tex (hunk 6 above)
Comment (wujastyk) [reply documentation/biblatex-manuscripts-philology.tex]: I mean like what biblatex calls a "block".  At present, there's a heading "Contents" and another "Remarks," each starting a new section of the layout.  That's what I mean.  Contents and Annotation fields are not run together in a single paragraph.

Comment (maieul) [reply documentation/biblatex-manuscripts-philology.tex]: in this case, I will use "bllock" terme. Section can refer to \section, or to \beginnumbering... we should not add a third use!

File: documentation/biblatex-manuscripts-philology.tex (hunk 3 above)
Comment (maieul) [reply documentation/biblatex-manuscripts-philology.tex]: for brackets, you can use \mkbibrackets, which will take account of nested brackets.

File: documentation/biblatex-manuscripts-philology.tex (hunk 4 above)
Comment (maieul) [reply documentation/biblatex-manuscripts-philology.tex]: not a problem, I will correct. This compilation problem is quite strange, anyway

File: documentation/biblatex-manuscripts-philology.tex (hunk 6 above)
Comment (maieul) [reply documentation/biblatex-manuscripts-philology.tex]: corrected

**Pull request merged**